### ISSUE TITLE
[Feature Flags] Notification: add feature flag content to Usage and Code tabs

### ISSUE THREAD

**Issue by laurenmrice** [carbon-design-system/carbon-website#4222]
## Acceptance criteria

Create a PR to add feature flag content sections to Notification on the following tabs: 
- Usage tab
- Code tab

-----

# Notification

## Usage tab
- [x] Add a "Feature flag" tag to the top of the page, right above the "Description". 
       *This may require dev help. Refer to the Figma spec.

`### Feature flags`
- [x] Add net new section
- [x] Add section heading
- [x] Add content from Figma file

## Code tab
- [x] Add a "Feature flag" tag to the React resource card.
       *This may require dev help. Refer to the Figma spec.

`### Feature flags`
- [x] Add net new section
- [x] Add section heading
- [x] Add content from Figma file

------

**Review**
- [x] Create PR
- [ ] PR reviews

**Comment by Kritvi-bhatia17** [comment]
This issue is on hold until Benjamin's Notifications PRs (#4218, #4243 and #4270 ) are merged.

**Comment by laurenmrice** [comment]
This is now ready to be worked on and is unblocked.
